tools/elide-build/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://dl.less.build/toolchains/gradle/gradle-8.3-bin.zip
```

Comment:
needs upgrade to `8.4-SNAPSHOT` (grab from latest branch or we can grab from rebase)

Reply:
updated to match the root project 👍